fronts-client/src/bundles/chefsBundle.ts

```diff
@@ -33,52 +35,69 @@ export const fetchChefs =
     dispatch(actions.fetchStart(ids));
     try {
       const chefs = await liveRecipes.chefs(params);
-      const chefsWithTags = chefs.results.filter(chef=>chef.contributorType==="Profile");
-      if(chefsWithTags.length==0) {
+      const chefsWithTags = chefs.results.filter(
+        (chef) => chef.contributorType === 'Profile'
+      );
+      if (chefsWithTags.length == 0) {
         dispatch(actions.fetchSuccess([]));
         return;
       }
 
-      dispatch(fetchChefsById(chefsWithTags.map(chef =>chef.nameOrId), 1,20,true));
+      dispatch(
+        fetchChefsById(
+          chefsWithTags.map((chef) => chef.nameOrId),
+          1,
+          20,
+          true
+        )
+      );
     } catch (e) {
       dispatch(actions.fetchError(e));
     }
   };
 
-export const fetchChefsById = (
-  tagIds: string[],
-  page = 1,
-  pageSize = 20,
-  alreadyStarted:boolean = false
-): ThunkResult<void> =>
+export const fetchChefsById =
+  (
+    tagIds: string[],
+    page = 1,
+    pageSize = 100,
+    alreadyStarted: boolean = false
+  ): ThunkResult<void> =>
   async (dispatch) => {
+    //we could be called as the second part of a two-stage fetch, OR we could be called directly.
+    //if called directly, indicate that we started a fetch. Otherwise, the first part should
+    if (!alreadyStarted) dispatch(actions.fetchStart(tagIds));
 
-  //we could be called as the second part of a two-stage fetch, OR we could be called directly.
-  //if called directly, indicate that we started a fetch. Otherwise, the first part should
-  if(!alreadyStarted) dispatch(actions.fetchStart(tagIds));
+    try {
+      const chefTags = await liveCapi.chefs({
+        ids: tagIds.join(','),
+        'show-elements': 'image',
+        'show-fields': 'all',
+        'page-size': pageSize,
```

Comment:
Keeping signature of this method as it is except pageSize to be  default to 100 as the suggested fix for maximum chefs to get loaded.